packages/dds/tree/src/test/shared-tree/fuzz/anchorStability.fuzz.spec.ts

```diff
@@ -31,36 +29,29 @@ import {
 	createAnchors,
 	deterministicIdCompressorFactory,
 	failureDirectory,
-	fuzzNode,
-	initialFuzzSchema,
 	validateAnchors,
+	type FuzzNode,
+	createOnCreate,
 } from "./fuzzUtils.js";
 import type { Operation } from "./operationTypes.js";
-import { typedJsonCursor } from "../../../domains/index.js";
+import type { NodeBuilderData } from "../../../internalTypes.js";
+// eslint-disable-next-line import/no-internal-modules
+import { jsonableTreeFromForest } from "../../../feature-libraries/treeTextCursor.js";
 
 interface AnchorFuzzTestState extends FuzzTestState {
 	// Parallel array to `clients`: set in testStart
 	anchors?: Map<Anchor, [UpPath, Value]>[];
+	initialJsonableTree?: JsonableTree[];
 }
 
-const config = {
-	schema: initialFuzzSchema,
-	// Setting the tree to have an initial value is more interesting for this targeted test than if it's empty:
-	// returning to an empty state is arguably "easier" than returning to a non-empty state after some undos.
-	get initialTree() {
-		return typedJsonCursor({
-			[typedJsonCursor.type]: fuzzNode,
-			sequenceChildren: [1, 2, 3],
-			requiredChild: {
-				[typedJsonCursor.type]: fuzzNode,
-				requiredChild: 0,
-				sequenceChildren: [4, 5, 6],
-			},
-		});
+const initialTreeState: NodeBuilderData<typeof FuzzNode> = {
```

Comment:
why aren't we able to type this as FuzzNode and use the FuzzNode constructor (passing in the same json)?

Reply:
Unfortunately since we are using `FuzzNode` for casting during schema comparisons, and creating different schema (which satisfies the structure of `FuzzNode`) in the fuzzViews (as during updates it creates a new schema), creating an instance of the `FuzzNode` leads to a `document out of schema` error

Reply:
This us super gross and unsafe, but I think justified. That said, I think it should get a comment explaining why it can't use unhydrated nodes, and also why is can't be type safe (needs that type cast at the end), since neither of those limitation are obvious and in almost all cases either of those would be bad.